src/lib/selector/Query.ts

```diff
@@ -6,9 +6,11 @@ export class Query {
 
     public static async executeQuery(connection: Connection, query: string, logLabel: string, recordsCount?: number) {
         if (recordsCount === undefined || recordsCount < 200) {
-            Util.showSpinner('-- Querying ' + logLabel);
+            const messageString = '-- Querying ' + logLabel;
+            Util.showSpinner(messageString);
+            const initialTabbing = (messageString.length > 22) ? (messageString.length > 30) ? (messageString.length > 38) ? (messageString.length > 46) ? '\t\t' : '\t\t\t' : '\t\t\t\t' : '\t\t\t\t\t' : '\t\t\t\t\t';
```

Comment:
Znowu ten initialTabbing, niech on będzie schowany wewnątrz tego Util'a. 
Funkcja `executeQuery` ma za zadanie wykonać query, nie ma sensu jej zaśmiecać logiką `pritifajowania` logów bo to nie jest jej odpowiedzialność żeby printować ładne logi